### ISSUE TITLE
SonarQube 9.2 support

### ISSUE THREAD

**Issue by evasokolova1f** [sonar-scala/sonar-scala#786]
Hi!
Are there plans to add support for SonarQube version 9.2?

**Comment by mwz** [comment]
Hi @evasokolova1f, I'm catching up on recent releases, but yes a release for 9.2 is planned.

**Comment by evasokolova1f** [comment]
Hi @mwz!
Can you please tell me when can I expect the release? Approximately will work too.

**Comment by mwz** [comment]
I'll try and take a closer look before the end of the year and let you know once I know more - I'm not sure atm whether there have been any breaking changes in the 9.x series.

**Comment by yarosman** [comment]
@mwz I can try to help with updating if you help me to resolve error locally
```
java.lang.NoSuchMethodError: 'scala.Option com.sksamuel.scapegoat.Inspection.explanation()'
```
when I try to compile project

**Comment by mwz** [comment]
Thanks @yarosman!

This doesn't happen for me and in the CI, so looks to me like it's related to your local environment. If you can open a separate issue with some more details on your project set up, which line you're getting this error from etc I can try and help you resolve this.

**Comment by saif-ellafi** [comment]
Hi all, @mwz 
Any updates perhaps on 9.2 support? Anyone tested it to see what happens? We can test it if such information is missing, or test any progress branch if direction is kind of clear.

**Comment by shafemoh** [comment]
Hi all,
Any timeliness for when a stable release of this plugin will be released and that will support Sonarqube EE v9.2.4?

Regards,
@Shafemoh

**Comment by mwz** [comment]
support for SonarQube 9.4 has been released thanks to @Fristi - sonar-scala [v9.0.0](https://github.com/sonar-scala/sonar-scala/releases/tag/v9.0.0)
you can get the artifact directly from Sonatype or use the docker images from https://hub.docker.com/r/mwizner/sonarqube-scala-plugins
